main.scss

```diff
@@ -2,11 +2,12 @@
 @import "o-colors/main";
 @import "o-typography/main";
 @import "o-share/main";
+$o-banner-is-silent: false;
```

Comment:
would it be possible to import in silent mode and apply the styles directly in the sass.  It would help reduce the stylesheet size